doc/news/changes/minor/20190107GiovanniAlzetta

```diff
@@ -0,0 +1,4 @@
+Modified/added: Now GridTools::build_global_description_tree does
+not require mpi and has a test.
+<br>
+(Giovanni Alzetta, 2019/01/07)
```

Comment:
I would say drop this file -- we have these changelog entries for users to see what changed from one version to another, but the change here is a follow-up to a previous one that happened after the previous release and so there is no change from one release to another that is not already covered by an existing changelog entry.

Reply:
I actually never fully understood these files, aren't there some "guidelines"? I mean, the idea is that they should be added for new functions/new features/changes which could modify the "user experience", so that anyone can keep track of what changed in the library? Anyway, just dropped it. Guess I shall do it also with the other PRs, where user-wise the function does not really change.